### DISCUSSION TITLE
Noto Emoji as fallback font works great -- but Noto Color Emoji does not work despite color rendering being on

### DISCUSSION THREAD

**Discussion by arooni** [koreader/koreader#11310] [General]
Device is a Samsung S23 Ultra.  The noto emoji font as a fallback works flawlessly but when I try to switch to color noto, no text shows up.  Should I open a bug report for this?

**Comment by NiLuJe** [comment]
I'm pretty sure we don't support color fonts ;).

EDIT: Yup, FT buffers are always Y8, and we always expect to be using grayscale AA.

I only have a vague idea of how color fonts work in FT, but that's definitely never going to happen if it requires specific buffer handling.

(There's also a few build-time dependencies that we might not fulfill, either).

**Reply by Frenzie** [reply]
Heh, I'd kind of expect a grayscale fallback or something.

**Comment by poire-z** [comment]
There had been some work in Coolreader upstream about color emoji fonts: https://github.com/buggins/coolreader/pull/289
But totally not interested in that and adding another layer of complexity to crengine.

**Reply by NiLuJe** [reply]
Right, I realized that what I mentioned above was actually from looking at our UI code; I imagine it'd be even crazier to deal with in CRe ;).
